v2.0/create-sequence.md

```diff
@@ -105,16 +148,39 @@ Insert a few records to see the sequence.
 +----+----------+--------------------+
 ~~~
 
-<!-- ### View the Current Value of a Sequence
+### View the Current Value of a Sequence
+
+To view the current value without incrementing the sequence, use:
 
 {% include copy-clipboard.html %}
 ~~~ sql
 > SELECT * FROM customer_seq;
 ~~~
 ~~~
-~~~ -->
++------------+---------+-----------+
+| last_value | log_cnt | is_called |
++------------+---------+-----------+
+|          3 |       0 |   true    |
++------------+---------+-----------+
+~~~
+
+{{site.data.alerts.callout_info}}The <code>log_cnt</code> and <code>is_called</code> columns are returned only for PostgreSQL compatibility – they are not stored in the database.{{site.data.alerts.end}}
```

Comment:
@jseldess, similarly, does this note provide enough info?

Reply:
This sounds great. One nit: I'd prefer a semicolon instead of a dash.

Reply:
Done.